astropy/units/tests/test_format.py

```diff
@@ -248,20 +248,22 @@ def _test_roundtrip_cds(unit):
             yield _test_roundtrip_cds, val
 
 
-def test_roundtrip_ogip():
-    def _test_roundtrip_ogip(unit):
-        a = core.Unit(unit.to_string('ogip'), format='ogip')
-        assert_allclose(a.decompose().scale, unit.decompose().scale, rtol=1e-2)
-        try:
-            b = core.Unit(unit.decompose().to_string('ogip'), format='ogip')
-        except ValueError:  # skip mag: decomposes into dex, unknown to OGIP
-            return
-        assert_allclose(b.decompose().scale, unit.decompose().scale, rtol=1e-2)
-
-    x = u_format.OGIP
-    for key, val in x._units.items():
-        if isinstance(val, core.UnitBase) and not isinstance(val, core.PrefixUnit):
-            yield _test_roundtrip_ogip, val
+_glob_ogip_vals = []
+for key in sorted(u_format.OGIP._units):
```

Comment:
Tried putting this inside `def setup_module(module)` but `pytest` somehow only collected the first value in list that way. Not something to worry about since I am just messing around to understand performance issue.